db/dev_seeds/sdg.rb

```diff
@@ -1,3 +1,23 @@
 section "Creating Sustainable Development Goals" do
   load Rails.root.join("db", "sdg.rb")
+
+  SDG::Goal.all.each do |goal|
+    goal.targets.each do |target|
+      title = "Title for default locale"
+      description = "Description for default locale"
+      rand(2..3).times do |n|
+        local_target = SDG::LocalTarget.create!(code: "#{target.code}.#{n}",
+                                                title: title,
+                                                description: description,
+                                                target: target)
+        random_locales.map do |locale|
+          Globalize.with_locale(locale) do
+            local_target.title = "Title for locale #{locale}"
+            local_target.description = "Description for locale #{locale}"
+            local_target.save!
+          end
+        end
+      end
+    end
+  end
```

Comment:
Hi @javierm, working on the development seeds I created some code to create random relations between  Relatables and Related SDG's, not sure if I should include here or maybe a better place could be the PR #4269 you are working on 🤔 . Here is the portion of code I'm talking about:

```
  relatables = [Debate.last, Proposal.last, Poll.last, Legislation::Process.last, Budget::Investment.last]
  relatables.each do |relatable|
    3.times do
      random_goal = SDG::Goal.sample
      random_target = random_goal.targets.sample
      random_local_target = random_target.local_targets.sample
      relatable.sdg_goals << random_goal if [true, false].sample
      relatable.sdg_targets << random_target if [true, false].sample
      relatable.sdg_local_targets << random_local_target if [true, false].sample
    end
  end
```

Do you have any preference about this?

Reply:
No preference :thinking:.